### PR TITLE
Add Maltiverse integration section

### LINKED ISSUE #6216: Add documentation for new Maltiverse integration
| Related issue |
|--|
|https://github.com/wazuh/wazuh/issues/17752|

## Description

A new integration with Maltiverse is being developed here:
- https://github.com/wazuh/wazuh/issues/17752

We will need to add a section explaining how to configure and use it, similarly to other existing integrations as Pagerduty, Slack, Virustotal, etc.

### PR DESCRIPTION
**Pull request by fdalmaup**
## Description
Closes #6216. It adds the `Maltiverse` native integration section in the `Integration with external APIs` page of the documentation and the corresponding `integration` reference. 

## Checks
- [x] Compiles without warnings.
- [x] Uses present tense, active voice, and semi-formal registry.
- [x] Uses short, simple sentences.
- [x] Uses **bold** for user interface elements, _italics_ for key terms or emphasis, and `code` font for Bash commands, file names, REST paths, and code.
- [x] Uses three spaces indentation.
- [x] Adds or updates meta descriptions accordingly.
- [x] Updates the `redirects.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).

### CODE REVIEW

**Review by GGP1 — APPROVED** [review]
The instructions, syntax and spelling is correct, the links are pointed to the right resources and the styling of the content matches the rest of the content

maltiverse

LGTM

File: source/user-manual/manager/manual-integration.rst [hunk 1]
```diff
@@ -179,6 +179,41 @@ Once the configuration is complete, alerts start showing in the email inbox.
...
+#. Edit ``/var/ossec/etc/ossec.conf`` in the Wazuh server and include a configuration block such as the following. Replace ``MALTIVERSE_AUTH_TOKEN`` with your Maltiverse Authentication Token. The rule level filter is optional. You can remove it or set another level value for the integration.
+
+   .. code-block:: xml
+      :emphasize-lines: 3
+
+      <integration>
+         <name>custom-maltiverse</name>
+         <hook_url>https://api.maltiverse.com</hook_url>
+         <level>3</level>
+         <api_key>MALTIVERSE_AUTH_TOKEN</api_key>
```
Comment (GGP1) [inline source/user-manual/manager/manual-integration.rst:200]: Just as a comment, the other sections add a comment where values should be replaced, like 

```
<api_key>API_KEY</api_key> <!-- Replace with your VirusTotal API key -->
```

I don't think it's necessary but perhaps we should add it to be consistent with the rest of the page

**Review by Selutario — CHANGES_REQUESTED** [review]

**Review by Selutario — APPROVED** [review]

File: source/user-manual/reference/ossec-conf/integration.rst [hunk 2]
```diff
@@ -116,15 +116,15 @@ This filters alerts by where the event originated.
...
+This writes the alert file in the JSON format. The Integrator makes use of this file to fetch fields values.
 
 +--------------------+-----------------------------------------------------------+
 | **Default value**  | n/a                                                       |
 +--------------------+-----------------------------------------------------------+
 | **Allowed values** | json                                                      |
 +--------------------+-----------------------------------------------------------+
 
-.. note:: This option must be set to ``json`` for Slack, VirusTotal and Shuffle integrations.
+.. note:: This option must be set to ``json`` for Slack, VirusTotal, Shuffle and Maltiverse integrations.
```
Comment (javimed) [inline source/user-manual/reference/ossec-conf/integration.rst:127]:
```suggestion
.. note:: This option must be set to ``json`` for Slack, VirusTotal, Shuffle, and Maltiverse integrations.
```

**Review by javimed — CHANGES_REQUESTED** [review]

File: source/user-manual/manager/manual-integration.rst [hunk 3]
```diff
@@ -179,6 +179,41 @@ Once the configuration is complete, alerts start showing in the email inbox.
    :align: center
    :width: 80%
 
+Maltiverse
+----------
+
+`Maltiverse <https://whatis.maltiverse.com/>`__ is an open and collaborative platform for indexing and searching Indicators of Compromise (IoCs). It works as a broker for Threat intelligence sources. Maltiverse aggregates information from more than a hundred different public, private and community sources. This integration identifies IoCs in Wazuh alerts via the Maltiverse API and generates new enriched ones inserting new fields, following the threat taxonomy of the ECS standard (Elastic Common Schema).
```
Comment (javimed) [inline source/user-manual/manager/manual-integration.rst:185]:
```suggestion
`Maltiverse <https://whatis.maltiverse.com/>`__ is an open and collaborative platform for indexing and searching Indicators of Compromise (IoCs). It works as a broker for Threat intelligence sources. Maltiverse aggregates information from more than a hundred different public, private and community sources.

This integration identifies IoCs in Wazuh alerts via the Maltiverse API. It generates new alerts enriched with Maltiverse data. The Maltiverse data fields are based on the threat taxonomy of the ECS standard (Elastic Common Schema).
```

File: source/user-manual/manager/manual-integration.rst [hunk 4]
```diff
@@ -179,6 +179,43 @@ Once the configuration is complete, alerts start showing in the email inbox.
    :align: center
    :width: 80%
 
+Maltiverse
+----------
+
+`Maltiverse <https://whatis.maltiverse.com/>`__ is an open and collaborative platform for indexing and searching Indicators of Compromise (IoCs). It works as a broker for Threat intelligence sources. Maltiverse aggregates information from more than a hundred different public, private and community sources.
+
+This integration identifies IoCs in Wazuh alerts via the Maltiverse API and generates new enriched ones inserting new fields, following the threat taxonomy of the ECS standard (Elastic Common Schema).
```
Comment (javimed) [inline source/user-manual/manager/manual-integration.rst:187]:
```suggestion
This integration identifies IoCs in Wazuh alerts via the Maltiverse API. It generates new alerts enriched with Maltiverse data. The Maltiverse data fields are based on the threat taxonomy of the ECS standard (Elastic Common Schema).
```

**Review by javimed — APPROVED** [review]
LGTM

**Pull request merged**